.github/copilot-instructions.md

```diff
@@ -0,0 +1,28 @@
+---
+applyTo: "**"
```

Comment:
Esto hace un apply to all? Mmm permíteme dudar, debería aplicar sólo a lo que realmente aplica, like ".go"

Reply:
Aplica a all por que si te fijas hay cosas tambien del Makefile por ej, a lo sumo tendria que separar las reglas para que algunas solo sean de .go

Reply:
SRP, para mí aplica también para estos casos

Reply:
![image](https://github.com/user-attachments/assets/714e28a6-71d4-475d-b62b-d78b528b418b)

A quien le compro que cosa?

Reply:
> SRP, para mí aplica también para estos casos

Decis entonces crear un instructions x caso de uso? bueno era la idea pero no encontre como hacerlo y los tire todos en uno jajaja